evmrpc/block.go

```diff
@@ -28,6 +28,7 @@ import (
 )
 
 const ShellEVMTxType = math.MaxUint32
+const TestGenesisBlockHash = "0xF9D3845DF25B43B1C6926F3CEDA6845C17F5624E12212FD8847D0BA01DA1AB9E"
```

Comment:
Why is this called TestGenesisBlockHash? Is this related to test at all?

Reply:
ah good catch. Not really, moved it back to where it was before and got rid of the "Test" name.